module/pom.xml

```diff
@@ -162,7 +167,31 @@
     </plugins>
   </build>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.mule</groupId>
+        <artifactId>mule-javaee-runtime-bom</artifactId>
+        <version>4.6.0-SNAPSHOT</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
   <dependencies>
+    <dependency>
+      <groupId>org.mule.runtime</groupId>
+      <artifactId>mule-extensions-api</artifactId>
+      <version>1.5.0</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.mule.runtime</groupId>
+      <artifactId>mule-api</artifactId>
+      <version>1.5.0</version>
+      <scope>provided</scope>
+    </dependency>
```

Comment:
We should review this. These dependencies are given by the runtime version and it would make sense to change the parent pom to do this, so all the API imported is consistent. Then, we override the minMuleVersion locally to avoid breaking compatibility, and validate the connector works fine against older versions of the runtime by using our MTF tests.

Reply:
I eliminated them, because now they are coming from the parent